src/components/FormComponents/NewService.js

```diff
@@ -24,6 +24,15 @@ export default function NewService({ onAddNewService, onOpenNewServiceForm }) {
           Stundensatz
           <Costinput currentCostsPerHour="default: 50" />
         </label>
+        <label>
+          Notiz(Optional):
+          <ServiceNote
+            aria-label="notes"
+            data-testid="servicenote"
+            name="notes"
+            placeholder="Für Notizen zur Dienstleistung."
+          ></ServiceNote>
```

Comment:
If you have an aria-role, you don't need the testid. [According to their website](https://testing-library.com/docs/queries/bytestid): _In the spirit of the guiding principles, it is recommended to use this only after the other queries don't work for your use case._ You should look through your tests again and only use `data-testid` when you can't query it differently.
```suggestion
          <ServiceNote
            aria-label="notes"
            name="notes"
            placeholder="Für Notizen zur Dienstleistung."
          />
```